canonical/ubuntu.com
# feat: /ceph redesign

## Done

- Applied redesign as per [doc](https://docs.google.com/document/d/1gpEjTcdJor2yTGfMz7T_xIfs7BtV6rbBPFYfNZFe52Y/edit?tab=t.0) and [design](https://www.figma.com/design/ulUyOFdQ7CNy0whuJ4S2FM/Ceph-Bubble?node-id=13-5965&node-type=frame&m=dev)
- Drive by:
  - Bump Vanilla version 
## QA

- View the site locally in your web browser at: https://ubuntu-com-14498.demos.haus/ceph
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-16343
